### PR TITLE
Implements PUT teams endpoint

### PR DESCRIPTION
**Pull request by Yserz**

### CODE REVIEW

File: src/ts/team/TeamAPI.ts [hunk 1]
```diff
@@ -28,6 +28,21 @@ export default class TeamAPI {
...
   }
 
+  public putTeam(team: TeamData): AxiosPromise {
+    const config: AxiosRequestConfig = {
+      data: {
+        name: team.name,
+        icon: team.icon,
+      },
+      method: 'put',
+      url: `${TeamAPI.URL.TEAMS}/${team.id}`
```
Comment (lipis) [inline src/ts/team/TeamAPI.ts]: @bennyn no linter for dangling comma?

File: src/ts/team/TeamAPI.ts [hunk 2]
```diff
@@ -28,6 +28,21 @@ export default class TeamAPI {
     });
   }
 
+  public putTeam(team: TeamData): AxiosPromise {
```
Comment (bennyn) [inline src/ts/team/TeamAPI.ts]: What will be the response from the server? Can you define it already in the method signature? 

Like: `public putTeam(team: TeamData): AxiosPromise<TeamData>` ?

File: src/ts/team/TeamAPI.ts (hunk 1 above)
Comment (bennyn) [inline src/ts/team/TeamAPI.ts]: There is no ESLint file in this project. You can commit one.

File: src/ts/team/TeamAPI.ts (hunk 2 above)
Comment (Yserz) [inline src/ts/team/TeamAPI.ts]: Basically nothing is returned

**Pull request merged**